service/service.go

```diff
@@ -138,3 +150,37 @@ func (s *Service) Boot() {
 		go s.KVMClusterConfigFramework.Boot()
 	})
 }
+
+func newBaseClusterConfig(f *flag.Flag, v *viper.Viper) (*cluster.Config, error) {
```

Comment:
Is this (and utility functions) ok to be here or would it make sense to split this to `cluster_config.go`?

Reply:
I think this does not belong here at all.

Reply:
Do you mean this should be put to another file or somewhere else totally? In earlier discussion it was said that Viper shouldn't escape service. That's why this is here.

Reply:
I was under the impression the base cluster config when injected was not used at all so I wonder why we do this here.